hepcrawl/spiders/edp_spider.py

```diff
@@ -0,0 +1,430 @@
+        node = get_node(response.meta["record"])
+        article_type = response.meta.get("article_type")
+
+        record = HEPLoader(item=HEPRecord(), selector=node, response=response)
+        if article_type in ['correction',
+                            'addendum']:
+            record.add_xpath('related_article_doi',
+                             ".//related-article[@ext-link-type='doi']/@href")
+            record.add_value('journal_doctype', article_type)
+
+        record.add_value('dois', response.meta.get("dois"))
+        record.add_xpath('page_nr', ".//counts/page-count/@count")
+        record.add_xpath('abstract', './/abstract[1]')
+        record.add_xpath('title', './/article-title/text()')
+        record.add_xpath('subtitle', './/subtitle/text()')
+        record.add_value('authors', self._get_authors(node))
+        record.add_xpath('collaborations', ".//contrib/collab/text()")
+
+        free_keywords, classification_numbers = self._get_keywords(node)
+        record.add_value('free_keywords', free_keywords)
+        record.add_value('classification_numbers', classification_numbers)
+
+        journal_title = node.xpath(
+            './/abbrev-journal-title/text()|//journal-title/text()').extract_first()
+        record.add_value('journal_title', journal_title)
+        record.add_xpath('journal_issue', './/issue/text()')
+        record.add_xpath('journal_volume', './/volume/text()')
+        record.add_xpath('journal_artid', './/elocation-id/text()')
+
+        fpage = node.xpath('.//fpage/text()').extract()
+        lpage = node.xpath('.//lpage/text()').extract()
+        record.add_value('journal_fpage', fpage)
+        record.add_value('journal_lpage', lpage)
+
+        published_date = self._get_published_date(node)
+        record.add_value('journal_year', published_date[:4])
+        record.add_value('date_published', published_date)
+
+        record.add_xpath('copyright_holder', './/copyright-holder/text()')
+        record.add_xpath('copyright_year', './/copyright-year/text()')
+        record.add_xpath('copyright_statement',
+                         './/copyright-statement/text()')
+        record.add_value('copyright_material', 'Article')
+        record.add_xpath('license', './/license/license-p/ext-link/text()')
+        record.add_xpath('license_type', './/license/@license-type')
+        record.add_xpath('license_url', './/license/license-p/ext-link/@href')
+
+        record.add_value('collections', self._get_collections(
+            node, article_type, journal_title))
+
+        if "pdf_links" in response.meta:
+            record.add_value(
+                "additional_files",
+                self._create_fft_file(
+                    get_first(response.meta["pdf_links"]),
+                    "INSPIRE-PUBLIC",
+                    "Fulltext"
+                )
+            )
+        record.add_value("urls", response.meta.get("urls"))
+
+        # FIXME: are the references somewhere?
```

Comment:
Yup. In harvesting kit are represented: https://github.com/inspirehep/harvesting-kit/blob/master/harvestingkit/edpsciences_package.py#L64